.travis.yml

```diff
@@ -37,19 +37,28 @@ install:
       source venv/bin/activate
       pip install cffi virtualenv pytest numpy
     fi
-  - cargo install --git https://github.com/PyO3/pyo3-pack.git --tag v0.6.1 pyo3-pack
   - rustup default nightly-2019-07-19
   - rustup component add rustfmt
   - rustup component add clippy
 
 script:
   - cargo fmt --all -- --check
   - cargo clippy -- -D warnings
-  - pyo3-pack develop
+  - |
```

Comment:
Small comment: the if/download/unpack parts should probably be in the install section.